sql-ballerina/tests/simple-params-query-test.bal

```diff
@@ -608,22 +609,22 @@ function queryTimestampStringInvalidParam() {
     record{}|error? result = trap queryMockClient(simpleParamsDb, sqlQuery);
     test:assertTrue(result is error);
 
-    if (result is ApplicationError) {
-        test:assertEquals(result.message(), "Error while executing SQL query: SELECT * from " +
-        "DateTimeTypes WHERE timestamp_type =  ? . Timestamp format must be yyyy-mm-dd hh:mm:ss[.fffffffff]");
+    if (result is DatabaseError) {
+        test:assertTrue(result.message().startsWith("Error while executing SQL query: SELECT * from DateTimeTypes " +
+        "WHERE timestamp_type =  ? . data exception: invalid datetime format."));
     } else {
-        test:assertFail("ApplicationError Error expected.");
+        test:assertFail("DatabaseError Error expected.");
     }}
 
-@test:Config {
-    groups: ["query", "query-simple-params"]
-}
-function queryTimestampLongParam() {
-    //1486122780000 : 2017-02-03 11:53:00
-    TimestampValue typeVal = new (1486122780000);
-    ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE timestamp_type = ${typeVal}`;
-    validateDateTimeTypesTableResult(queryMockClient(simpleParamsDb, sqlQuery));
-}
+//@test:Config {
+//    groups: ["query", "query-simple-params"]
+//}
+//function queryTimestampLongParam() {
+//    //1486122780000 : 2017-02-03 11:53:00
+//    TimestampValue typeVal = new (1486122780000);
+//    ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE timestamp_type = ${typeVal}`;
+//    validateDateTimeTypesTableResult(queryMockClient(simpleParamsDb, sqlQuery));
```

Comment:
same here

Reply:
removed in https://github.com/ballerina-platform/module-ballerina-sql/pull/161/commits/1a363a5654c30b1c3b4efd836eb9225713c4e5a6